static/src/stylesheets/amp/_hosted.scss

```diff
@@ -0,0 +1,848 @@
+    font-weight: bolder;
+    margin: 0;
+}
+
+.hosted__next-page--client-name {
+    font-size: 20px;
+    line-height: 20px;
+    margin: 0;
+}
+
+.hosted__next-page--tile {
+    display: block;
+    color: #333333;
+    text-decoration: none;
+    position: relative;
+    min-height: 56px;
+    margin: 10px 0;
+    border-top: 2px solid #e0e0e0;
+    box-sizing: border-box;
+}
+
+.hosted-next-page__wrapper {
+    background: #ffffff;
+}
+
+.hosted__next-page-thumb {
+    height: 54px;
+    width: $thumb-width;
+    object-fit: cover;
+    position: absolute;
+    margin-right: 6px;
+}
+
+.hosted__next-page-title {
+    margin: 0 0 0 $thumb-width;
+}
+
+.hosted__header {
+    @include f-textSans;
+    width: 100%;
+    height: $header-height;
+    background-color: #333333;
+    position: relative;
+}
+
+.has-no-flex .hosted__header .hostedbadge {
+    position: absolute;
+    top: 0;
+    margin-top: 30px;
+}
+
+.hosted__headerwrap {
+    margin: 0 auto;
+    text-align: right;
+    height: 100%;
+    position: relative;
+    display: flex;
+}
+
+.hostedbadge {
+    align-self: flex-end;
+    position: relative;
+    margin-left: 20px;
+    width: $badge-width;
+    z-index: 1015;
+}
+
+.hostedbadge:focus {
+    text-decoration: none;
+}
+
+.hostedbadge--pushdown {
+    top: 0;
+}
+
+.hostedbadge__info {
+    font-size: 14px;
+    font-weight: 600;
+    letter-spacing: .5px;
+    line-height: 16px;
+    height: auto;
+    padding-top: 5px;
+    padding-bottom: 8px;
+    color: #333333;
+    margin: 0;
+    text-align: center;
+}
+
+.hostedbadge .hostedbadge__logo {
+    position: absolute;
+    display: block;
+    width: 100%;
+    top: 100%;
+    background-color: #ffffff;
+}
+
+.hosted-header__spacer {
+    flex: 1;
+}
+
+.hosted__video video {
+    width: 100%;
+}
+
+.hosted-page {
+    .content__article-body {
+        margin: 0;
+    }
+
+    .content__article-body .media-primary {
+        margin-left: -10px;
+    }
+
+    .u-responsive-ratio img {
+        width: 100%;
+        height: 100%;
+        position: absolute;
+        top: 0;
+        left: 0;
+    }
+
+    .content .gs-container {
+        padding: 0 10px;
+    }
+}
+
+.hosted-article-page {
+    .content-article-body {
+        margin: 0;
+    }
+
+    .media-primary.media-primary--showcase {
+        position: relative;
+        height: 378px;
+        margin-bottom: 10px;
+        background-size: cover;
+        background-position: center center;
+        &::after {
+            content: '';
+            display: block;
+            position: absolute;
+            top: 0;
+            left: 0;
+            width: 100%;
+            height: 100%;
+            z-index: 2;
+            background: linear-gradient(to top, rgba(51, 51, 51, .4) 0%, rgba(51, 51, 51, .4) 2%, transparent 100%);
+        }
+    }
+
+    &.hosted-page--bright .content__article-body p.intro + p::first-letter {
+        color: #ffffff;
+    }
+
+    .gs-container {
```

Comment:
I think we normally advise against adding things to `gs-container`, is this needed and or can we put this on something else?